src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -69,6 +77,16 @@ static ConnectionProvider newConnection() {
 		return NewConnectionProvider.INSTANCE;
 	}
 
+	static Builder elastic(){
```

Comment:
When creating the `ConnectionProvider` the `name` is not optional. It is used for the metrics.
Once the builder approach is introduced the methods below for creating `ConnectionProvider` will be deprecated/removed.